src/svgeditor/BitmapEdit.as

```diff
@@ -136,10 +136,11 @@ public class BitmapEdit extends ImageEdit {
 	}
 
 	protected override function selectHandler(evt:Event = null):void {
```

Comment:
For the sake of readability, could I get you to put a line at the top like `var foo:ObjectTransformer = currentTool as ObjectTransformer;` (with a better name than `foo` of course) and then simplify the rest of the function? I know you're not the one that started the fire but it seems like it's continuing to spread ;)